google/resource_container_cluster.go

```diff
@@ -508,8 +528,8 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 	d.Partial(true)
 
 	// The master must be updated before the nodes
-	if d.HasChange("master_version") {
-		desiredMasterVersion := d.Get("master_version").(string)
+	if d.HasChange("min_master_version") {
+		desiredMasterVersion := d.Get("min_master_version").(string)
```

Comment:
This doesn't take into consideration if the current `master_version` is already greater than `min_master_version`. I think we want to ignore the version in that case, instead of downgrading?

Reply:
Good point, fixed.